src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -444,15 +242,15 @@
         :style                             {:background-color (colors/theme-colors colors/white
                                                                                    colors/neutral-95
                                                                                    theme)}
-        :content-container-style           {:padding-top distance-from-last-message}
+        :content-container-style           {:padding-top    distance-from-last-message
+                                            :padding-bottom top-margin}
         :inverted                          true
         :on-layout                         #(on-layout
```

Comment:
this is werid, will this be improved later? we shouldn't use anon functions, and passing all those atoms

Reply:
I didn't rework the whole chat screen, I tried to focus my changes/improvements to just the areas needed (for example, in this file, I just touched the header), also I kept the same behavior.

Ofc, it can be revisited and improved, probably in another issue, so the review is easier, wdyt?

Reply:
yes, sure ,thank you